website/package.json

```diff
@@ -14,6 +16,9 @@
   },
   "dependencies": {
     "@aragon/ui": "^1.0.0",
+    "@aragon/wrapper": "^5.0.0-rc.26",
```

Comment:
Is it possible to use a preview version of the toolkit instead for transaction pathing? Even if it is not super stable in terms of API I think its fine for us if we just pin to a very specific version, given that the transaction pathing is sort of mature for this use case.

Reply:
Right now, there's no preview version of the toolkit available for doing this (I tried), but we will migrate this to toolkit as soon as it's available.